kernel/genlambda.mli

```diff
@@ -51,8 +50,7 @@ and 'v lam_branches =
 and 'v fix_decl = Name.t Context.binder_annot array * 'v lambda array * 'v lambda array
 
 type evars =
-    { evars_val : constr evar_handler;
-      evars_metas : metavariable -> types }
+  { evars_val : constr evar_handler }
```

Comment:
Why not 
```suggestion
  constr evar_handler
```
?

Reply:
No, I prefer to keep a record because it makes the closure explicit. Also it's more future-proof.

Reply:
But evar_handler is already a record?
Also future proofing barely ever works

Reply:
Hm, right, evar_handler is indeed a record already. But I'd still argue for future proofing because it decouples the type of evar expansion from the kernel from the one here which could be more general.